app/src/main/java/com/easyrouteapp/MapActivity.java

```diff
@@ -0,0 +1,114 @@
+package com.easyrouteapp;
+
+import android.content.DialogInterface;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.Toast;
+
+import com.easyrouteapp.async.ReverseGeocodingTask;
+import com.easyrouteapp.event.GeoCodeErrorEvent;
+import com.easyrouteapp.event.RefreshStartLoadingEvent;
+import com.easyrouteapp.event.RefreshStopEvent;
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.MapFragment;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.model.LatLng;
+
+import org.greenrobot.eventbus.EventBus;
+import org.greenrobot.eventbus.Subscribe;
+import org.greenrobot.eventbus.ThreadMode;
+
+public class MapActivity extends AppCompatActivity implements OnMapReadyCallback, GoogleMap.OnMapLongClickListener {
+    private static final String TAG_LOG = "[MapActivity]";
+    public static final String EXTRA_DEFAULT_MAP_NAME = "EXTRA_DEFAULT_MAP_NAME";
+    private LatLng defaultCoordinates;
+    private GoogleMap googleMap;
+    protected SwipeRefreshLayout mSwipeRefreshLayout;
```

Comment:
why you are using swipe to refresh layout if you are not refreshing anything while swiping? (for all the screens)